exercises/chapter5/test/Main.purs

```diff
@@ -111,9 +111,8 @@ Note to reader: Delete this line to expand comment block -}
           $ shapeText $ Line origin { x: 1.0, y: 1.0 }
     suite "Exercise Group - Newtype" do
       test "Exercise - calculateWattage" do
-        let
-          (MakeWatt w) = calculateWattage (Amp 0.5) (Volt 120.0)
-        Assert.equal 60.0 w
+        Assert.equal (MakeWatt 60.0)
+          $ calculateWattage (Amp 0.5) (Volt 120.0)
```

Comment:
I think this is a little more idomatic

Reply:
I think I tried something like that, but as I recall, the compiler complained about the lack of an Eq instance for `Watt`. If this works, I'm good with it.